tests/acceptance/pageObjects/FilesPageElement/offCanvasPage.js

```diff
@@ -0,0 +1,63 @@
+const util = require('util')
+
+module.exports = {
+  commands: {
+    getMediaViewer: function() {
+      return this.useXpath()
+        .waitForElementVisible('@mediaviewer')
+        .click('@mediaviewer')
+        .waitForElementVisible('@actionBar')
+        .useCss()
+    },
+    openMediaViewer: function(fileName) {
```

Comment:
The function openMediaViewer just waits for it to be loaded. Maybe you can rename it to something like `waitForMediaViewerLoaded`